content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -152,6 +152,18 @@ CNI-Genie also supports [assigning multiple IP addresses to a pod](https://githu
 
 [Contrail](http://www.juniper.net/us/en/products-services/sdn/contrail/contrail-networking/), based on [OpenContrail](http://www.opencontrail.org), is a truly open, multi-cloud network virtualization and policy management platform. Contrail / OpenContrail is integrated with various orchestration systems such as Kubernetes, OpenShift, OpenStack and Mesos, and provides different isolation modes for virtual machines, containers/pods and bare metal workloads.
 
+### DANM
+
+[DANM](https://github.com/nokia/danm) is a networking solution for telco workloads running in a Kubernetes cluster. It's built up from the following components:
+
+   * a CNI plugin capable of provisioning IPVLAN interfaces with advanced features
```

Comment:
Suggestion: Start each bullet with an upper-case letter. That will be consistent with most of the other lists in the doc set.

* A CNI plugin capable of ...